content/03-how-tos/06-How-to-contribute-to-the-Mantis-roadmap.md

```diff
@@ -0,0 +1,25 @@
+---
+title: How to Contribute to the Mantis Roadmap
+metaTitle: How to contribute to the Mantis roadmap
+---
+# How to Contribute to the Mantis Roadmap
+
+## Introduction
+
+The Mantis team at IOHK is hard at work developing exciting new features for the Mantis client, and you are welcome to add your contribution.
+
+## Ways you can help
+
+The Mantis team welcomes offers of help. 
+
+Writing code is valuable, but new code can be problematic to integrate into the existing code base.
```

Comment:
Should link to the relevant codebases:
- https://github.com/input-output-hk/mantis/
- https://github.com/input-output-hk/mantis-wallet/

And perhaps clarify again that it is open source software?